components/profile_selector.js

```diff
@@ -71,7 +71,6 @@ const mapDispatchToProps = dispatch => {
 
 const mapStateToProps = reduxState => {
   return {
-    multipleChoiceOptions: reduxState.multipleChoiceOptions,
     reduxState: reduxState,
     profileQuestions: reduxState.questions.filter(
       q => q.variable_name !== "needs"
```

Comment:
Should you also filter out the feedback questions as you do below?

Reply:
This component actually isn't being used any more... so I just made an issue for deleting it: #1812